src/test/java/com/google/api/codegen/protoannotations/testdata/ruby_library_no_gapic_config.baseline

```diff
@@ -2772,50 +2772,38 @@ module Library
         self::GRPC_INTERCEPTORS = LibraryServiceClient::GRPC_INTERCEPTORS
       end
 
-      SHELF_PATH_TEMPLATE = Google::Gax::PathTemplate.new(
-        "shelves/{shelf_id}"
-      )
-
-      private_constant :SHELF_PATH_TEMPLATE
-
-      DELETED_BOOK_PATH_TEMPLATE = Google::Gax::PathTemplate.new(
-        "_deleted-book_"
+      ARCHIVED_BOOK_PATH_TEMPLATE = Google::Gax::PathTemplate.new(
```

Comment:
This is just re-ordering of the resource name artifacts. i thought it would make more sense to order them alphabetically, as it's deterministic and easy to reason about.

Reply:
How were they ordered previously? Is this going to cause churn when we switch a client from v1 to v2 config?

Reply:
Pretty sure it was ordered by the order in GAPIC interface's `collections`, followed by any top-level non-duplicate API `collections`

So if there's no `collections` field in the GAPIC config v2 to order the resource names, yeah we'll most likely get churn

Reply:
OK. Not sure whether there is a coherent alternative, but I think that avoiding churn is a worthwhile goal. If it's not possible, we may just need to do a regeneration ahead of time to get the churn out of the way.

Reply:
I think we would have to do the regeneration pre-party, yeah. We could either draw ire now by changing the current generator to order resource name artifacts alphabetically (a billion changes at once for Yoshi), or API-by-API inside GAPIC configs (manual work on our part, but diffs are spread over time for Yoshi).